Eat_and_Greet/comps/Toolbar/CornerButton.tsx

```diff
@@ -1,8 +1,11 @@
 import React from 'react';
 import {View, TouchableOpacity, Text, Alert, StyleSheet} from 'react-native';
 import styles from '../../style';
+import { useNavigation } from '@react-navigation/native';
 
-const CornerButton = ({name = 'default name', top = 0, left = 0, bottom = 0, right = 0, width = "80%", height = "10%", alertText}) => {
+const CornerButton = ({name = 'default name', top = 0, left = 0, bottom = 0, right = 0, width = "80%", height = "10%", onPressFunc}) => {
```

Comment:
this can be made to look a bit cleaner, it seems it bit much:
`interface TextBoxProps {
  placeholder: string;
}`

Reply:
can you elaborate on what you mean?